cli/azd/pkg/auth/azd_credential.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package auth
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/public"
+)
+
+var _ azcore.TokenCredential = &azdCredential{}
+
+type azdCredential struct {
+	client  *public.Client
+	account *public.Account
```

Comment:
The manager has a `newCredential` instance method that just constructs the object with the `&<type-name>{}` syntax.  But it would be clearer if I just had newAzdCredential method.  I'll fix this.